libs/utils/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["Centrifuge <admin@centrifuge.io>"]
+description = 'Utility functions for Centrifuge'
+edition = '2021'
+license = "LGPL-3.0"
+name = 'cfg-utils'
+repository = "https://github.com/centrifuge/centrifuge-chain"
```

Comment:
In some crates of our workspace, the link continues with something like `/libs/utils`, but not sure what should be the correct one

Reply:
Yeah I noticed that when I copy-pasted this cargo file. Those links are all 404 and I don't see the need to make them super specific to the sub-crate, just pointing to the repo itself is fine. I fixed that here: https://github.com/centrifuge/centrifuge-chain/pull/1086